source/lac/trilinos_precondition2.cc

```diff
@@ -0,0 +1,339 @@
+// ---------------------------------------------------------------------
```

Comment:
Since this file is logically split, please call this file trilinos_precondition_ml.cc